cico_setup.sh

```diff
@@ -137,6 +137,23 @@ function deploy() {
   echo 'CICO: Image pushed, ready to update deployed app'
 }
 
+function deploy_snapshot() {
```

Comment:
Maybe refactor the `deploy` function above and not copy 90%?

Reply:
I just tried that here: 645f890ed0346b0d01472fe7e59269bc881a2f6e